DEVELOPER_GUIDE.md

```diff
@@ -75,6 +76,20 @@ testClusters {
 }
 ```
 
+##### Option 2
+Add the experimental feature flag as a command line argument
+- `./bin/opensearch -E opensearch.experimental.feature.extensions.enabled=true` when running from a local distribution
+- `./gradlew run -Dopensearch.experimental.feature.extensions.enabled=true` when running using gradle in developer mode
+
+##### Option 3
+Enable the experimental feature flag by setting it to true in `opensearch.yml`.
+- cd in to your local distribution build for OpenSearch.
+- now cd into config folder and open `opensearch.yml` in your local editor
+- Search for `opensearch.experimental.feature.extensions.enabled`, uncomment and set it to `true`
+- Run OpenSearch using 
+  - `./bin/opensearch` when running from a local distribution
+  - `./gradlew run` when running using gradle in developer mode
```

Comment:
I don't think so we can run `./gradlew run` for this option. For gradle run we need to add the feature flag in run.gradle which is already covered in option 1

Reply:
I still haven't seen an example of the not-working run.gradle file.  That's how I'm running it and it works.

Reply:
true

Reply:
@dbwiddis I think I know where I got confused. After modifying run.gradle, I tried running the assembled version with `./bin/opensearch` as mentioned here thinking that the feature flag will be copied over. But that is not the case. run.gradle modification works only with `./gradlew run` and not `./bin/opensearch`

In order to run it with `./bin/opensearch` you need to either configure the feature flag via opensearch.yml or via command-line argument